include/mbedtls/pk_info.h

```diff
@@ -106,6 +106,14 @@ struct mbedtls_pk_info_t
      * usage restrictions into account. */
     int (*can_do)( const void * ctx, mbedtls_pk_type_t type );
 
+    /** Signature size
+     *
+     * mbedtls_pk_signature_size() returns this value.
```

Comment:
The documentation for that function should be updated too, so that the user knows what a return value of `0` means.

Reply:
Done